src/components/LeftNavigation/LeftNavigation.Props.ts

```diff
@@ -24,6 +24,7 @@ export interface ILeftNavigationOption {
     selected?: boolean;
     disabled?: boolean;
     position?: LeftNavigationOptionPositionEnum;
+    notificationNumber?: number;
```

Comment:
could this work with ie. any label? say i wanted to add the 'New' text above an icon?

Reply:
no, the point is that it is a number. this is only bubble for notifications. and this needs to be a number so that I can check if number is bigger then 10, then only 9+ is shown.